src/features/MoviesList/styled.js

```diff
@@ -1,39 +1,39 @@
-import { Link } from "react-router-dom";
 import styled from "styled-components";
 
-export const Wrapper = styled.ul`
+export const List = styled.ul`
   display: grid;
   gap: 24px;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-columns: repeat(4, 1fr);
   margin: 0;
   padding: 0;
   list-style-type: none;
 
-  & > li {
-    transition: transform 0.15s ease-out;
+  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
+    gap: 16px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    grid-template-columns: 1fr;
   }
+`;
+
+export const Item = styled.li`
+  transition: transform 0.15s ease-out;
 
-  & > li:hover {
+  &:hover {
     transform: translateY(-12px);
   }
 
   @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
-    gap: 16px;
-    grid-template-columns: 1fr 1fr;
-    & > li:hover {
+    &:hover {
```

Comment:
Good idea with the media query, you can immediately see the style text at low resolution, we won't come back to that anymore.